docs/source/usage.rst

```diff
@@ -24,41 +24,33 @@ Preparing your repository
 to determine how to build it. It is philosophically similar to
 `Heroku Build Packs <https://devcenter.heroku.com/articles/buildpacks>`_.
 
-Locating and composing configuration files
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-``repo2docker`` will look for configuration files **located** in two places:
+In general, ``repo2docker`` uses configuration files that are already part of
+various data science workflows (e.g., ``requirements.txt``), rather than
+creating new custom configuration files.
 
-* A folder called ``binder`` in the root of the repository.
-* The root of the repository. (If a folder called ``binder`` exists in the root
-  of the repository, configuration files outside of that folder will be ignored.)
+``repo2docker`` configuration files are all **composable** - you can use any number
+of them in the same repository.
 
-.. note::
+Locating and composing configuration files
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-   In general, ``repo2docker`` uses configuration files that are already part of
-   various data science workflows (e.g., ``requirements.txt``), rather than
-   creating new custom configuration files.
+``repo2docker`` will look for configuration files located in two places:
 
-``repo2docker`` configuration files are all **composable** - you can use any number
-of them in the same repository. 
+* A folder named ``binder`` in the root of the repository.
+* The root of the repository.
 
 There are a few notable rules for composition precedence and build priority:
 
-* ``Dockerfile``: If a Dockerfile is present in a repository, it will take precedence
-  over all other configuration files (which will be ignored).
-* ``environment.yml`` with ``requirements.txt``: If both of these files are
-  present, then ``environment.yml`` will be used to build the image, **not**
-  ``requirements.txt``. If you wish to ``pip install`` packages using an
-  ``environment.yml`` file, you should do so with the
+* If the folder ``binder`` is the root
```

Comment:
"is the root directory" -> "If a folder named `binder/` is lcoated at the top leve lof the repository only configuration files in it will be considered"?